### ISSUE TITLE
RenderComponent.Label

### ISSUE THREAD

**Issue by EtienneBruines** [EngoEngine/engo#136]
The `RenderComponent` has a `string` variable called `Label`. It is currently not in use, but it can be set. 

Do we want to use it? Or is it only there for the user?

**Comment by paked** [comment]
@EtienneBruines In the past, that was used for linking `Resource`s and `RenderComponent`s, without having to initialise a resource. It seems legacy now, but maybe this feature is something we should look into?

**Comment by EtienneBruines** [comment]
I'd say we remove it. Thoughts?

**Comment by paked** [comment]
I'm doing this now.

**Comment by EtienneBruines** [comment]
Fixed in c3ef5c834a4ce7377cc21f36dd6c917252b955ab / #206.

**Comment by mewmew** [comment]
The function signature of `NewRenderComponent` should probably be updated to reflect this change.

Before:
```go
func NewRenderComponent(d Drawable, scale Point, label string) RenderComponent
```

After:
```go
func NewRenderComponent(d Drawable, scale Point) RenderComponent
```
